model/src/main/java/uk/gov/hmcts/payment/api/service/RemissionServiceImpl.java

```diff
@@ -107,7 +107,6 @@ public Remission createRetrospectiveRemissionForPayment(RetroRemissionServiceReq
         PaymentFeeLink paymentFeeLink = populatePaymentFeeLink(paymentGroupReference);
         PaymentFee fee = populatePaymentFee(feeId, paymentFeeLink, remissionServiceRequest);
         FeePayApportion feePayApportion = populatePaymentApportionment(feeId);
```

Comment:
We can remove this FeePayApportion method implementation and call. Since we are not dealing with Payment in new requirement.

Reply:
Refactored the code